src/classify/classifier/svm_wrapper.cpp

```diff
@@ -96,9 +114,16 @@ void svm_wrapper::train(const std::vector<doc_id>& docs)
         out << idx_->liblinear_data(d_id) << "\n";
     out.close();
 
+#ifndef _WIN32
     std::string command = svm_path_ + executable_ + "train "
                           + options_.at(kernel_) + " svm-train";
     command += " > /dev/null 2>&1";
+#else
+    // see comment in classify()
+    auto command = "\"\"" + svm_path_ + executable_ + "train.exe\" "
```

Comment:
They are generated by the ExternalProject calls in the CMakeLists.txt in the classify dir, just like they are on Linux. All executables have ".exe" added automatically (including ours), since Windows doesn't run anything without that extension.